game.py

```diff
@@ -86,6 +98,23 @@ def pull_card_from_deck(self, player):
         print(f"{player.name} pulled {pulled_card} from the deck ({len(self.deck)} cards left in the deck) \n")
         player.cards.append(pulled_card)
 
+    def block_the_next_player(self):
+        self.advance_to_next_player_turn()
+        self.notify_player_was_blocked()
+
+    def block_initial_player(self):
+        self.notify_player_was_blocked()
```

Comment:
have the order of these methods the same as in block_the_next_player()
it may seem to not matter but it could cause confusion in debugging